lib/teracy-dev/location/git_synch.rb

```diff
@@ -92,6 +110,8 @@ def sync(location, sync_existing)
           Dir.chdir(lookup_path) do
             @logger.info("cd #{lookup_path} && git clone #{git_remote_url} #{dir}")
             system("git clone #{git_remote_url} #{dir}")
+
+            abort_process_if_fail_to_clone git_remote_url, dir
```

Comment:
Why this step only available in first git clone? what if the authenticate fail after first clone? (for example user changed password)

Reply:
if first clone success, if will cached your credentials, so when git pull or git fetch, we will no need to auth anymore

if the authenticate fail after first clone has success, maybe user no longer able to use git clone/pull, we should have this case, but I don't think it will always happen